server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -187,6 +190,15 @@ export class DocumentStorage implements IDocumentStorage {
             lastSummarySequenceNumber: 0,
         };
 
+        const sessionP: ISession = {
```

Comment:
Here too..we probably can remove the P suffix?

Reply:
Thanks. Done. :)